README.md

```diff
@@ -917,39 +921,91 @@ end
 
 ### Rake tasks
 
-#### `chewy:update` and `chewy:reset`
+For a Rails application, some index-maintaining rake tasks are defined.
 
-Inside the Rails application, some index-maintaining rake tasks are defined.
+#### `chewy:reset`
+
+Rerforms zero-downtime reindexing as described [here](https://www.elastic.co/blog/changing-mapping-with-zero-downtime). So the rake task creates a new index with uniq suffix and then simply aliases it to the common index name. The previous index is deleted afterwards (see `Chewy::Index.reset!` for more details).
 
 ```bash
 rake chewy:reset # resets all the existing indices, declared in app/chewy
 rake chewy:reset[users] # resets UsersIndex only
-rake chewy:reset[users,projects] # resets UsersIndex and ProjectsIndex
-rake chewy:reset[-users,projects] # resets every index in application except specified ones
+rake chewy:reset[users,places] # resets UsersIndex and PlacesIndex
+rake chewy:reset[-users,places] # resets every index in application except specified ones
```

Comment:
It's not very obvious for me: does it skip both users and places? I'd suggest to use `[-users,-places]` syntax then